opm/simulators/wells/StandardWellPrimaryVariables.cpp

```diff
@@ -475,6 +484,10 @@ volumeFraction(const unsigned compIdx) const
     else if (FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx) &&
              FluidSystem::phaseIsActive(FluidSystem::gasPhaseIdx)) {
         well_fraction -= evaluation_[GFrac];
+
+        if constexpr (Indices::enableSolvent) {
```

Comment:
I think we also only need one the following code? 

```c++
        if constexpr (Indices::enableSolvent) {
            well_fraction -= evaluation_[SFrac];
        }
```

Maybe it might cause problem for single phase solvent case?